chatkit/src/main/java/com/pusher/chatkit/ChatManager.kt

```diff
@@ -103,23 +115,31 @@ class ChatManager @JvmOverloads constructor(
 
     private fun ChatKitEvent.handleEvent() {
```

Comment:
Lower case `k` in Chatkit 😉

Reply:
Gonna rename it to `ChatEvent` for now (will be user state machine in the future)

Reply:
`ChatManagerEvent` as `ChatEvent` is taken